config/webpack.common.js

```diff
@@ -474,13 +450,31 @@ module.exports = function (options) {
        */
       new LoaderOptionsPlugin({}),
 
-      extractCSS,
+      // ExtractTextPlugin
```

Comment:
Are these commented lines intentional?

Reply:
yes, as I try to work out how to use it but don't want to lose what I was doing